### ISSUE TITLE
Please everyone read! Open manifesto to the velveye.

### ISSUE THREAD

**Issue by Fredyfaruk** [ValveSoftware/Dota2-Gameplay#20614]
### Description

Friends, Valve cheats on some accounts. The account where this is most obvious is my own account and the comments I have reviewed. What is this cheat? They say that Candy Shop gives you random rewards, but it is a lie. You may have seen it on the internet, the guy drops 3 arcanas in 50 rerolls. Sometimes 2... but people who don't care filled their inventory with arcanas. But some accounts, one of which is my account. There is no arcana even in 600 rerolls. Forget arcanas, there is not even any small reward. Continuous ward courier and 10-year sets. Now Valve is clearly lying to us. This is definitely not luck. There are some restrictions in the algorithm (which they do not disclose) and those restrictions do not allow some accounts to give big rewards. Maybe because of their nationality or because they say that those with a lot of stuff will give money anyway, they ensure that the big reward drops to new accounts. But there is definitely a cheat. Please put pressure on Valve to explain the algorithm and say why they did this. Create public opinion... because we are strong together.

### Example Match ID (and possibly Timestamp)

2216512651651

### Screenshots

![image](https://github.com/user-attachments/assets/f9b23636-f376-4b32-b844-f43903cf951f)

**Comment by Fredyfaruk** [comment]
up up

**Comment by Jordan11599** [comment]
Non-issue. You need to stop, this Github isn't for you to post this on. Nothing will be done.
